charts/urlaubsverwaltung/README.md

```diff
@@ -64,107 +61,72 @@ It is used for the following values:
 It is important that these values be configured as strings. Otherwise, installation will fail.
 See example for [Using external Database](#using-external-database).
 
-
 ### JVM Settings
 
 Urlaubsverwaltung sets the following system properties by default:
 `-XX:+PrintFlagsFinal`
 
-You can override thes by setting the `JAVA_OPTS` environment variable.
+You can override these by setting the `JAVA_OPTS` environment variable.
 
 ```yaml
 extraEnv: |
-  - name: JAVA_OPTS
-    value: >-
-      -XX:+PrintFlagsFinal
-      -Djava.net.preferIPv4Stack=true
-      -Djava.awt.headless=true
+    - name: JAVA_OPTS
+      value: >-
+        -XX:+PrintFlagsFinal
+        -Djava.net.preferIPv4Stack=true
+        -Djava.awt.headless=true
 ```
 
-
 ### Database Setup
 
-By default, Bitnami's [MariaDB](https://github.com/bitnami/charts/tree/master/bitnami/mariadb) chart is deployed and used as database.
-Please refer to this chart for additional MariaDB configuration options.
-
+By default, Bitnami's [PostgreSQL](https://github.com/bitnami/charts/tree/master/bitnami/postgresql) chart is deployed
+and used as database.
+Please refer to this chart for additional PostgreSQL configuration options.
 
 #### Using external Database
 
 ```yaml
 extraEnv: |
     - name: spring.datasource.url
-      value: jdbc:mariadb://external-database:3306/urlaubsverwaltung
+      value: jdbc:postgresql://external-database:5435/urlaubsverwaltung
 
 extraEnvFrom: |
     - secretRef:
         name: '{{ include "urlaubsverwaltung.fullname" . }}-database'
 
 secrets:
-  database:
-    stringData:
-      spring.datasource.username: '{{ .Values.db.username }}'
-      spring.datasource.password: '{{ .Values.db.password }}'
+    database:
+        stringData:
+            spring.datasource.username: '{{ .Values.db.username }}'
+            spring.datasource.password: '{{ .Values.db.password }}'
 ```
 
-
 ### Authentication
 
-
-#### LDAP
-
-```yaml
-extraEnv: |
-    - name: UV_SECURITY_AUTH
-      value: ldap
-    - name: UV_SECURITY_DIRECTORYSERVICE_LDAP_URL
-      value: "ldap://ldap.example.org/"
-    - name: UV_SECURITY_DIRECTORYSERVICE_LDAP_BASE
-      value: dc=example,dc=org
-    - name: MANAGEMENT_HEALTH_LDAP_ENABLED
-      value: "false"
-
-extraEnvFrom: |
-  - secretRef:
-      name: '{{ include "urlaubsverwaltung.fullname" . }}-ldap-manager'
-
-secrets:
-  ldap-manager:
-    stringData:
-      UV_SECURITY_DIRECTORYSERVICE_LDAP_MANAGER_DN: '{{ .Values.ldap.manager.dn }}'
-      UV_SECURITY_DIRECTORYSERVICE_LDAP_MANAGER_PASSWORD: '{{ .Values.ldap.manager.password }}'
-```
-
-###### LDAP Sync
+#### OpenID Connect
 
 ```yaml
 extraEnv: |
```

Comment:
You shouldn't do this as a string, it should at least be an array like it will be in the deployment

Even better would be an object with the key being the variable name